lib/phoenix/endpoint/server.ex

```diff
@@ -30,6 +29,7 @@ defmodule Phoenix.Endpoint.Server do
   defp default(config, otp_app, port) do
```

Comment:
shall I better test this function asserting for the default value of `config` at `:handler` ?
would `Kernel#apply` work for testing private functions?